src/Microsoft.SqlTools.ServiceLayer/ExecutionPlan/ExecPlanGraph/executionplangraph.cs

```diff
@@ -6,12 +6,12 @@
 using System;
 using System.Collections.Generic;
 
-namespace Microsoft.SqlTools.ServiceLayer.ShowPlan.ShowPlanGraph
+namespace Microsoft.SqlTools.ServiceLayer.ExecutionPlan.ExecPlanGraph
 {
     /// <summary>
     /// Extension of graph with some handy included methods specific for ShowPlan use
     /// </summary>
-    public class ShowPlanGraph : Graph
+    public class ExecutionPlanGraph : Graph
```

Comment:
Please avoid changing the names of the internal classes ported from SSMS. It will make future code porting and cross referencing a little difficult.